src/models/validators/CommonSchema.ts

```diff
@@ -123,7 +123,7 @@ export const testTypesCommonSchema = baseTestTypesCommonSchema.keys({
   customDefects: Joi.array()
     .items(
       Joi.object().keys({
-        referenceNumber: Joi.string().max(10).required(),
+        referenceNumber: Joi.string().max(10).optional(),
```

Comment:
Can you add a test with an appropriate non-car IVA payload that has additionalDefects, but is missing the referenceNumber please.